### PR TITLE
Working AU CocoaUI for DPF

### PR DESCRIPTION
**Pull request by zamaudio**
Current output of auval:

```
VERIFYING CUSTOM UI
Carbon View Components Available: 0

GetPropertyInfo asked for CocoaUI
GetPropertyInfo asked for CocoaUI
GetProperty asked for CocoaUI
Cocoa Views Available: 1
ERROR: Can't Create Bundle from Cocoa View Bundle URL
file://localhost/Library/Audio/Plug-Ins/Components/d_parameters.component/Contents/MacOS/d_parameters

```

### CODE REVIEW

**Comment by zamaudio** [comment]
Install path for PLUGIN_DSP.component -> `/Library/Audio/Plug-Ins/Components`

Install path for PLUGIN_UI.bundle -> `$(HOME)/Library/Bundles`

File: Makefile.plugins.mk [hunk 1]
```diff
@@ -76,14 +78,21 @@ AU_BUILD_FLAGS = \
...
 # FIXME no-overloaded-virtual should be fixed!
 
-AU_LINK_FLAGS = \
+AU_DSP_LINK_FLAGS = \
 	-bundle \
 	-framework AudioToolbox \
 	-framework AudioUnit \
 	-framework CoreFoundation \
 	-framework CoreServices \
+	-framework Cocoa \
```
Comment (falkTX) [inline Makefile.plugins.mk]: why cocoa in the DSP side?

Comment (zamaudio) [reply Makefile.plugins.mk]: Typo

**Comment by zamaudio** [comment]
Above force-pushed commit detects the UI correctly

**Comment by zamaudio** [comment]
The reason that the above failed CI is because DSSI UI was attempted to be built with direct access enabled. 
```
# error DSSI UIs do not support direct access!
```

**Pull request closed without merging**